array_schema.go

```diff
@@ -425,3 +426,22 @@ func (a *ArraySchema) Type() (ArrayType, error) {
 
 	return ArrayType(arrayType), nil
 }
+
+// Handle a LoadArraySchemaRequest. This is used by TileDB-Cloud
```

Comment:
The convention for go doc is to start with the entity name like `HandleLoadArraySchemaRequest .....`